depends/hosts/darwin.mk

```diff
@@ -82,3 +82,6 @@ darwin_debug_CFLAGS=-O1 -g
 darwin_debug_CXXFLAGS=$(darwin_debug_CFLAGS)
 
 darwin_cmake_system_name=Darwin
+# Darwin version, which corresponds to OSX_MIN_VERSION.
+# See https://en.wikipedia.org/wiki/Darwin_(operating_system)
+darwin_cmake_system_version=20.1
```

Comment:
Where/how does CMake translate this to `OSX_MIN_VERSION`? Ideally we'd compose these somehow.